demo/src/main/java/com/eis/geoCalendar/demo/MapsActivity.java

```diff
@@ -1,85 +1,54 @@
 package com.eis.geoCalendar.demo;
 
+import android.os.Bundle;
+
 import androidx.core.app.ActivityCompat;
 import androidx.fragment.app.FragmentActivity;
 
-import android.location.Location;
-import android.os.Bundle;
-import android.widget.Toast;
-
-import com.eis.geoCalendar.app.GenericEvent;
 import com.eis.geoCalendar.app.GenericEventManager;
+import com.eis.geoCalendar.demo.Behaviour.EventMapBehaviour;
+import com.eis.geoCalendar.demo.Localization.LocationManager;
 import com.eis.geoCalendar.events.Event;
 import com.eis.geoCalendar.events.EventManager;
-import com.eis.geoCalendar.gps.GPSPosition;
-import com.google.android.gms.maps.CameraUpdateFactory;
-import com.google.android.gms.maps.GoogleMap;
-import com.google.android.gms.maps.GoogleMap.OnMapLongClickListener;
-import com.google.android.gms.maps.GoogleMap.OnInfoWindowLongClickListener;
-import com.google.android.gms.maps.OnMapReadyCallback;
 import com.google.android.gms.maps.SupportMapFragment;
-import com.google.android.gms.maps.model.LatLng;
-import com.google.android.gms.maps.model.Marker;
-import com.google.android.gms.maps.model.MarkerOptions;
-import com.eis.geoCalendar.demo.Localization.LocationManager;
-import com.eis.geoCalendar.demo.Localization.Command;
+
 
 
 /**
  * @author Turcato
  */
-public class MapsActivity extends FragmentActivity implements OnMapReadyCallback,
-        OnMapLongClickListener, OnInfoWindowLongClickListener, ResultEventListener, RemoveEventListener {
+public class MapsActivity extends FragmentActivity {
 
-    private GoogleMap mMap;
     private EventManager<Event<String>> eventManager;
+
+
     private static final String CREATE_EVENT_DIALOG_TAG = "createEventDialog";
     private static final String REMOVE_EVENT_DIALOG_TAG = "removeEventDialog";
     private static final int APP_PERMISSION_REQUEST_CODE = 0;
     private LocationManager locationManager;
+    private EventMapBehaviour eventMapBehaviour;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'eventMapBehaviour' could be replaced by a local variable.](https://app.codacy.com/manual/CremaLuca/geoCalendar/pullRequest?prid=4951239)